Core/source/test.ts

```diff
@@ -349,28 +352,29 @@ ava.default('parseKeys', async t => {
           { "fingerprint": "9EF2F8F36A841C0D5FAB8B0F0BAB9C018B265D22", "longid": "0BAB9C018B265D22", "shortid": "8B265D22", "keywords": "ARM FRIEND ABOUT BIND GRAPE CATTLE" }
         ],
         "algo": {
-          "algorithm": "rsa_encrypt_sign",
-          "bits": 2048,
+          "algorithm": "rsaEncryptSign",
+          "bits": 2047,
```

Comment:
Also worrying, probably related to the key change above, which should be reverted.